README.md

```diff
@@ -375,6 +375,29 @@ have read pemissions to the GCS bucket you would like to use.
 defaults to `dump`. This may be extended in future to support other formats
 such as `csv`, `avro` etc.
 
+`host` Specifies the host name for the source database.
+If not specified in case of direct connection to the source database, HarbourBridge
+fetches it from the environment variables.
+
+`user` Specifies the user for the source database.
+If not specified in case of direct connection to the source database, HarbourBridge
+fetches it from the environment variables.
+
+`dbName` Specifies the name of the source database.
+If not specified in case of direct connection to the source database, HarbourBridge
+fetches it from the environment variables.
+
+`port` Specifies the port for the source database.
+If not specified in case of direct connection to the source database, HarbourBridge
+fetches it from the environment variables.
+
+`password` Specifies the password for the source database.
+If not specified in case of direct connection to the source database, HarbourBridge
+fetches it from the environment variables.
+
+`streamingCfg` Optional flag. Specifies the file path for streaming config.
+Please note that streaming migration is only supported for MySQL and Oracle databases currently.
+
```

Comment:
Since we add that HarbourBridge can fetch this information from env variables, can we add a link here to the README file which talks about environment variable usage in HarbourBridge?